client.go

```diff
@@ -120,6 +120,7 @@ func (c *Client) doRequest(method, uri string, body interface{}) ([]byte, error)
 		}
 
 		req.Header.Set("Content-Type", "application/json")
+		req.Header.Set("User-Agent", "aiven-go-client/"+version())
```

Comment:
Apparently static checkers are OK with this but I haven't seen `+` being used without surrounding spaces in Go code. I'd add those here (same for the other place using `version()`).